File: app/src/main/java/com/example/atilayorunmobillium/ui/MainActivity.java
package com.example.atilayorunmobillium.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.atilayorunmobillium.R;

import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/example/atilayorunmobillium/ui/MainActivity.kt
package com.example.atilayorunmobillium.ui

import dagger.hilt.android.AndroidEntryPoint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.atilayorunmobillium.R

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}